<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWin</class>
 <widget class="QMainWindow" name="ChatWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="chatView">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>37</y>
      <width>591</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="messageEdit">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>530</y>
      <width>499</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>504</x>
      <y>530</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_settings">
    <property name="geometry">
     <rect>
      <x>514</x>
      <y>3</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_menu">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>3</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
